packages/ckeditor5-easy-image/src/cloudservicesuploadadapter.ts

```diff
@@ -17,31 +18,31 @@ import { FileRepository } from 'ckeditor5/src/upload';
  *
  * After enabling this adapter you need to configure the CKEditor Cloud Services integration through
  * {@link module:cloud-services/cloudservices~CloudServicesConfig `config.cloudServices`}.
- *
- * @extends module:core/plugin~Plugin
  */
 export default class CloudServicesUploadAdapter extends Plugin {
 	/**
 	 * @inheritDoc
 	 */
-	static get pluginName() {
+	public static get pluginName(): 'CloudServicesUploadAdapter' {
 		return 'CloudServicesUploadAdapter';
 	}
 
 	/**
 	 * @inheritDoc
 	 */
-	static get requires() {
+	public static get requires(): PluginDependencies {
 		return [ 'CloudServices', FileRepository ];
 	}
 
+	private _uploadGateway?: UploadGateway;
```

Comment:
We used to define properties at the top of the class.

Reply:
Also maybe we could use:

```suggestion
	private _uploadGateway!: UploadGateway;
```

because it's initialized in `init()`?